cmd/abc/abc.go

```diff
@@ -104,9 +107,56 @@ func setLogEnvVars() {
 	}
 }
 
+// checkForUpdates asynchronously checks for updates and prints results to
+// returned buffer.
+// A sync.WaitGroup is returned so main can wait for completion before exiting.
+func checkForUpdates(ctx context.Context) func() {
+	updatesCh := make(chan string, 1)
+
+	go func() {
+		defer close(updatesCh)
+
+		var b bytes.Buffer
+		if err := abcupdater.CheckAppVersion(ctx, &abcupdater.CheckVersionParams{
+			AppID: version.Name,
+			// Version: version.Version,
+			Version: "0.4.0", // intentionally older than cur version to test
```

Comment:
I wonder what will happen if we're running an abc that was built from HEAD (so there's no semver version)? Should we skip the version check in that case?

Reply:
Added a check to skip